client/public/locales/en.json

```diff
@@ -15,8 +15,14 @@
     },
     "confirmDelete": "Are you sure you want to delete this automation ?",
     "buttons": {
-        "cancel": "Cancel"
+        "edit": "Edit",
+        "delete": "Delete",
+        "trigger": "Trigger",
+        "create": "Create an automation"
     }
-
+  },
+  "buttons": {
+    "cancel": "Cancel",
+    "confirm": "Confirm"
```

Comment:
we said we had to put the text in lowercase and it is in the attributes of the css that we change the letter to capital